### PR TITLE
A re-write and reorg of the top index page

### PR DESCRIPTION
**Pull request by thothmoses**
A re-write and reorg of the top index page.

References #xxxx

### CODE REVIEW

**Review by rstento — CHANGES_REQUESTED** [review]

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -5,22 +5,43 @@ include::ROOT:partial$include.adoc[]
 
 Welcome to the _Akka Platform Guide_!
 
-Akka is a toolkit for building highly concurrent, distributed, and resilient message-driven applications.
+This guide intends to help you get started with the array of technologies provided with the Akka Platform. Akka is a toolkit for building highly concurrent, distributed, and resilient message-driven applications.
```
Comment (rstento) [inline docs-source/docs/modules/ROOT/pages/index.adoc:8]: As a personal nit, I don't like use of the word "intends" in this context. It indicates good intentions that might not be fulfilled. :-)

Also, I think this is the place to begin introducing the different audiences and the difference between Akka Platform and Akka Cloud Platform. Maybe something like:

This guide provides information on using the Akka Platform and Akka Cloud Platform for developers, administrators, and dev/ops. The Akka Platform is a toolkit for building highly concurrent systems that are scalable, highly efficient, and resilient--using Reactive Priniciples. The Akka Cloud Platform adds tooling for deployment on Kubernetes and is available from Amazon Web Services (AWS) and (GCP).

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -5,22 +5,43 @@ include::ROOT:partial$include.adoc[]
 
 Welcome to the _Akka Platform Guide_!
```
[the remark this answers is not in the document]
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:6]: Yeah, I'll kill it.

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -5,22 +5,43 @@ include::ROOT:partial$include.adoc[]
...
 
-The tutorials in this guide show end-to-end use-cases implemented in Akka with the Reactive principles in mind. The tutorials go through implementing a system step-by-step to illustrate how the Akka APIs apply together.
+The tutorial structure in this guide follows end-to-end use-cases implemented in Akka with the Reactive principles in mind. The tutorials go through implementing a system step-by-step to illustrate how the Akka APIs apply together.
 
-== How to get the most out of this guide
+== Akka Cloud Platform
+Akka Cloud Platform helps to leverage Akka and take it to the next level. It is a subscription that enables you to quickly build and deploy cloud-native Microservices on your preferred cloud service provider. Akka Cloud Platform provides frameworks and runtimes for building cloud-native applications that easily integrate with other services in your chosen ecosystem.
```
[the remark this answers is not in the document]
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:15]: I'll Inject an Akka Platform description section. Truncate the lead para, and add breakout the info on ways of taking advantage.

File: docs-source/docs/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:8]: That looks much better

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,26 +1,60 @@
...
+
+*Why use Akka Cloud Platform?*
+
+* Satisfy easy-to-deploy and operational needs for Development and SRE teams
+* Reduce implementation time for integrating with cloud services
+* Zero business barriers for getting started
+
+*What does Akka Cloud Platform provide?*
+
+* Simple deployment and scaling of Kubernetes-powered Akka applications
```
Comment (rstento) [inline docs-source/docs/modules/ROOT/pages/index.adoc:38]: Some of these bullets are related to each other. Suggest:

* Integrated Telemetry - integrates with popular monitoring tools and provides visibility into your distributed application
* Ops/DevOps tooling - simple deployment and scaling of Kubernetes-powered Akka applications
* Simple Integration ...
* Available through popular Cloud Marketplaces
* * Pay-as-you-go or custom pricing
* * Integrated legal agreement, purchasing and billing

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,26 +1,60 @@
...
 
 * The challenges that Reactive Architecture can help you overcome
 * A summary of Reactive Principles
 * The benefits of using Akka
 
+== Cloud deployment
+
+Each section of the tutorial provides for both running locally or deploying to the cloud. The cloud deployment steps use the Akka Cloud Platform. If you are working through the cloud deployment steps, we recommend reviewing the xref:deployment:index.adoc[Akka Cloud Platform] section of this guide and the details for xref:deployment:cloud-deployment.adoc[] and your specific platform.
+
+Kubernetes has become the de-facto platform for deploying in the cloud. The Cloud deployment page provides an overview of how the Kubernetes deployment works with the Akka Cloud Platform. Each cloud deployment section of the tutorial details how to run in Kubernetes using the Akka Cloud Platform.
```
Comment (rstento) [inline docs-source/docs/modules/ROOT/pages/index.adoc:57]: I think you can remove the Kubernetes sentence. Doesn't make sense here.

**Review by rstento — APPROVED** [review]
Getting there! The one thing missing is that people can deploy applications built with Akka anywhere they choose. It doesn't have to be Kubernetes, and it can be on their own data centers or in the cloud. But this guide is all about deploying on Kubernetes. I'm approving so that you can get the other reviewers involved. I don't see that they've posted any comments yet.

File: docs-source/docs/modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:38]: Cool - tweaking

File: docs-source/docs/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,26 +1,60 @@
...
+
+
+== Akka Cloud Platform
+Akka Cloud Platform enhances Akka by adding operational tooling, services, and support. It is easily accessible from marketplaces, such as AWS and GCP.
+
+*Why use Akka Cloud Platform?*
+
+* Satisfy easy-to-deploy and operational needs for Development and SRE teams
+* Reduce implementation time for integrating with cloud services
+* Zero business barriers for getting started
```
[the remark this answers is not in the document]
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:34]: Make it parallel with the other two.
Get started without business barriers

File: docs-source/docs/modules/ROOT/pages/index.adoc (hunk 5 above)
Comment (thothmoses) [reply docs-source/docs/modules/ROOT/pages/index.adoc:57]: I am redacting most of this but leveraging your comment above to provide context.

**Comment by thothmoses** [comment]
@patriknw If you are ok with these changes, could you merge Main into Current so that Guide is live for the Monday launch?

**Review by ignasi35 — APPROVED** [review]
I left a comment re the `ROOT/index.adoc` but the rest looks good to me and I'd like to push this forward.

I think we can address my comment on the `ROOT/index.adoc` in a separate PR.

**Review by andreaTP — APPROVED** [review]

**Pull request merged**